app/src/main/java/org/groundplatform/android/ui/map/gms/features/LineStringRenderer.kt

```diff
@@ -72,6 +78,13 @@ constructor(private val resources: Resources, private val bitmapUtil: BitmapUtil
       jointType = JointType.ROUND
       zIndex = POLYLINE_Z
     }
+    updateTooltipMarker(geometry, map, tooltipText)
```

Comment:
Should we add a check for null text to prevent showing empty markers?

Reply:
handled internally to TooltipMarkerRenderer